src/lib/model/ng-artifacts-factory.ts

```diff
@@ -30,8 +31,8 @@ export class NgArtifactsFactory {
       main: this._unixPathJoin(rootPathFromSelf, 'bundles', `${this._makeUmdPackageName(ngPkg)}.umd.js`),
```

Comment:
Is `rootPathFromSelf` guaranteed to have forward slashes `/`?

```
const rootPathFromSelf: string = path.relative(ngPkg.sourcePath, ngPkg.rootSourcePath);
```

I can only check this in the next week when being back at the office Windows machine.

Reply:
It is not guaranteed to have forward slashes, but the `this._unixPathJoin` function should replace windows with posix path separators